test/units/testsuite-74.escape.sh

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: LGPL-2.1-or-later
+#
```

Comment:
Not a matter, but why an empty comment here?

Reply:
Whoops, that's a typo, fixed.